src/pages/usages/Responsive.js

```diff
@@ -1,110 +1,232 @@
+                <code>
+                  &nbsp;&nbsp;@media screen and (min-width:992px){`{`}
+                </code>
+                <br />
+                <code>&nbsp;&nbsp;&nbsp;&nbsp;background-color: #333333;</code>
+                <br />
+                <code>&nbsp;&nbsp;{`}`}</code>
+                <br />
+                <code>{`}`}</code>
+              </p>
+            </div>
           </div>
-        </div> */}
-				<blockquote className='docs-info'>
-					<strong>Note:</strong> Some of the <code>$break</code> value are
-					<code>xs, sm, md, lg, xl.</code> These will be apply for both
-					<code>mq()</code> and <code>mqmax()</code> mixin's <code>$break</code>
-					value.
-				</blockquote>
-
-				{/* <div className="content-block">
+          <h4>Example</h4>
+          <div className="example-row">
+            <p className="mb-1x">
+              <small>Resize window to view responsive effects.</small>
+            </p>
+            <div className="eg-box mobile-first-example"></div>
+          </div>
+          <div className="mb-4x">
+            <Syntax language="scss">{mqExample}</Syntax>
+          </div>
+        </div>
+
+        {/* non mobile first example with max-width */}
+        <div className="content-block">
+          <h4>Non-mobile first approach</h4>
+          <p className="page__brief">
+            This approach uses <b className="fw-600">max-width</b> media
+            condition. For non-mobile first approach{" "}
+            <b className="fw-600">mqmax($break)</b> mixin is provided. The usage
+            of the mixin and the generated output is show below. Here{" "}
+            <code>$break</code> is a required argument where RWD breakpoint
+            values should be provided.
+          </p>
+
+          <div className="row mb-4x">
+            <div className="col-6">
+              <p className="page_brief">
+                <b className="fw-500">Mixin input:</b>
+                <br />
+                <code>.item{`{`}</code>
+                <br />
+                <code>&nbsp;&nbsp;@include mqmax(992px){`{`}</code>
+                <br />
+                <code>&nbsp;&nbsp;&nbsp;&nbsp;background-color: #333333;</code>
+                <br />
+                <code>&nbsp;&nbsp;{`}`}</code>
+                <br />
+                <code>{`}`}</code>
+              </p>
+            </div>
+            <div className="col-6">
+              <p className="page_brief">
+                <b className="fw-500">Mixin output:</b>
+                <br />
+                <code>.item{`{`}</code>
+                <br />
+                <code>
+                  &nbsp;&nbsp;@media screen and (max-width:992px){`{`}
+                </code>
+                <br />
+                <code>&nbsp;&nbsp;&nbsp;&nbsp;background-color: #333333;</code>
+                <br />
+                <code>&nbsp;&nbsp;{`}`}</code>
+                <br />
+                <code>{`}`}</code>
+              </p>
```

Comment:
Donot use `&nbsp`. We have syntax highlighter package. Plz refer to previous files.